contents/tutorials/hogql-date-time-filters.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Using HogQL for advanced time and date filters
+date: 2023-05-19
+author: ["ian-vanagas"]
+showTitle: true
+sidebar: Docs
+featuredImage: ../images/tutorials/banners/tutorial-9.png
+tags: ["hogql", "insights"]
+---
+
+Because there are infinite ways to break down time, there are infinite ways to filter based on time. HogQL unlocks more of these in PostHog, and in this tutorial, we go through examples of how to use do that.
```

Comment:
```suggestion
Because there are infinite ways to break down time, there are infinite ways to filter based on time. HogQL unlocks more of these in PostHog, and in this tutorial we'll go through examples of how to use do that.
```